defaults/api_ents.go

```diff
@@ -0,0 +1,51 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docker/libentitlement/entitlement"
+	"github.com/docker/libentitlement/secprofile"
+)
+
+const (
+	engineDomain = "engine"
+
+	// FIXME: we need to define the possible domain, identifier and value for the entitlement.
+	APIEntID = engineDomain + ".api"
```

Comment:
Since `api` would be the entitlement's domain and `engine` the `subdomain` and `swarm` the `ID`, it'd make more sense to have `api` at the beginning I think and the following format:
`docker run --entitlements="api:engine.swarm=all:deny"`

WDYT?

Reply:
Sounds good!